doc/source/numpy_2_0_migration_guide.rst

```diff
@@ -92,27 +92,29 @@ geterrobj               Use the np.errstate context manager instead.
 Inf                     Use ``np.inf`` instead.
 Infinity                Use ``np.inf`` instead.
 infty                   Use ``np.inf`` instead.
-issctype
+issctype                Use ``isinstance(rep, (type, np.dtype))`` instead.
 issubclass\_            Use ``issubclass`` builtin instead.
 issubsctype             Use ``np.issubdtype`` instead.
 mat                     Use ``np.asmatrix`` instead.
-maximum_sctype
+maximum_sctype          Use a specific dtype instead. You should avoid relying
+                        on any implicit mechanism and select the largest dtype of
+                        a kind explicitly in the code.
 NaN                     Use ``np.nan`` instead.
 nbytes                  Use ``np.dtype(<dtype>).itemsize`` instead.
 NINF                    Use ``-np.inf`` instead.
 NZERO                   Use ``-0.0`` instead.
 longcomplex             Use ``np.clongdouble`` instead.
-longfloat               Use ``np.longdouble`` instead.      
+longfloat               Use ``np.longdouble`` instead.
 lookfor                 Search NumPy's documentation directly.
-obj2sctype
+obj2sctype              Use ``np.dtype(obj)`` instead.
```

Comment:
```suggestion
obj2sctype              Use ``np.dtype(obj).type`` instead.
```

It looks like `obj2sctype` returns `dtype.type` in all non-error cases.

Reply:
That's correct - I updated the migration guide and expired attrs message.